src/main/java/io/vlingo/actors/Grid.java

```diff
@@ -189,6 +190,7 @@ public void setNodeId(final Id nodeId) {
   }
 
   private Mailbox maybeRemoteMailbox(final Address address, final Definition definition, final Mailbox maybeMailbox, final Runnable out) {
+    // TODO consider quorum check and retry here (maybe suspend for QuorumAchieved?)
```

Comment:
@VaughnVernon this seems to be an excellent place to add the quorum check for actors starting locally on the grid. I had the idea of suspending the mailbox instead of blocking with retries. We can then implement `QuorumControl` for something that resumes the suspended mailboxes on `quorumAchieved`. Do you think that that could work?